model-files/runtime/mtctm2.properties

```diff
@@ -90,7 +90,18 @@ TNC.baseFare = 2.20
 TNC.costPerMile = 1.33
 TNC.costPerMinute = 0.24
 TNC.costMinimum = 7.20
-   
+
+#Toggle and controls for managing transit crowding assignments
```

Comment:
Where are these settings used?

Reply:
I have to defer to @jfdman. They were in the develop branch mtctm2.properties file already.

Reply:
I meant the lines below, the transit crowding lines.  I see now that they're read by the python script -- that's what I was wondering.  Are they only used by that script?  Since most (all?) of the properties in this file are read by java code in the core, I think it'd be helpful to add a short comment noting where these properties are used.  Thank you!

Reply:
Got it. I added a block of comments to this section of the properties file indicating how the attributes are used and what the attributes are.